Library/Homebrew/rubocops/keg_only.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "rubocops/extend/formula"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class KegOnly < FormulaCop
+        def audit_formula(node, _class_node, _parent_class_node, body_node)
+          keg_only_node = find_node_method_by_name(body_node, :keg_only)
+          return unless keg_only_node
+
+          offending_node(node)
```

Comment:
@vidusheeamoli  Please pass `keg_only_node` instead of `node` for the correct node to get highlighted.

Reply:
`find_node_method_by_name` sets `@offensive_node` and `@offense_source_range` so this may actually be removable entirely.

Reply:
Thank you for the feedback @GauthamGoli @MikeMcQuaid! 😃